lib/ex_image_info.ex

```diff
@@ -125,9 +125,10 @@ defmodule ExImageInfo do
     :pnm
   ]
 
-  @typedoc "The supported image formats."
+  @typedoc "Supported image format types."
   @type image_format ::
           :jpeg
+          | :jpg
```

Comment:
I am thinking if really removing this alias... as it is not standard and is redundant. Originally, someone wanted to have that alias, and I "pleased" him. However, I don't see a real reason to keep it.

Reviewing the [case](https://github.com/Group4Layers/ex_image_info/issues/1), dev was doing something really risky in many ways (`to_atom(ext)`).

However, I see the convenience of trying to check if the binary provided along with a filename-extension (string type), is "enough" valid. However, I see that as a bad pattern, extensions shouldn't be trusted.

And if they want to do something like that, they can just provide the alias on their side.

Thoughts?

Reply:
Well, these formats do _look_ like the file extension, so it's more a force of habit to short hand it to `jpg`. Thank you DOS for three letter file extensions. But if the downstream consumer is using dialyzer this should be caught.

One solution is to explicitly take the `:jpg` and emit a warning that it was used and that it will be removed.

Reply:
Only downside of accepting 2, is that `seems?/1` returns only `:jpeg`, not `:jpg`. 

Another alternative to not do anything, was to remove :jpeg, so `seems?/1` accepts/returns the same for a given jpeg file.

I have to say I have started thinking about this with the ISOBMFF tedious tests, as it has many extensions, and found files being heif-sequence but not using .heifs extension, or being an heif file, but using .heic extension, etc. Well, not to say capital letters (.HEIC, .BMP, that I find sometimes as well -  that only affecting the above case).